docs/src/snap/explanation/epa.md

```diff
@@ -0,0 +1,548 @@
+# Enhanced Platform Awareness
+
+Enhanced Platform Awareness (EPA) represents a methodology and a set of
+enhancements across various layers of the orchestration stack, aimed at
+optimising platform capability, configuration, and capacity usage. For
+telecommunications service providers using virtual network functions (VNFs) to
+deliver services, EPA offers enhanced application performance, increased
+input/output throughput, and improved determinism.
+
+EPA focuses on three key objectives: discovering, scheduling, and isolating
+server hardware capabilities. This document provides a detailed description of
+EPA capabilities and its integration into {{product}}. These capabilities are:
+
+-  **HugePage support**: In GA from Kubernetes v1.14, this feature enables the
+   discovery, scheduling, and allocation of HugePages as a first-class
+   resource.  
+-  **Real-time kernel**: Ensures that high-priority tasks are run within a
+   predictable time frame, providing the low latency and high determinism
+   essential for time-sensitive applications.  
+-  **CPU pinning** (CPU Manager for Kubernetes (CMK)): In GA from Kubernetes
+   v1.26, provides mechanisms for CPU pinning and isolation of containerised
+   workloads.  
+-  **NUMA Topology Awareness**: Ensures that CPU and memory allocation are
+   aligned according to the NUMA architecture, reducing memory latency and
+   increasing performance for memory-intensive applications.  
+-  **Single Root I/O Virtualization (SR-IOV)**: Enhances networking by enabling
+   virtualisation of a single physical network device into multiple virtual
+   devices.  
+-  **DPDK (Data Plane Development Kit)**: A set of libraries and drivers for
+   fast packet processing, designed to run in user space, optimising network
+   performance and reducing latency.
+
+This document provides detailed instructions for setting up and installing the
+aforementioned technologies. It is designed for developers and architects
+looking to integrate these new technologies into their {{product}}-based
+networking solutions. The goal is to achieve enhanced network I/O,
+deterministic compute performance, and optimised server platform sharing.
```

Comment:
yes. Thats because the 'original' doc was six times as long as this, and does also include setting them up, which I am instead going to spin-off into linked howto guides. Will reword